scripts/auto_release/main.py

```diff
@@ -452,6 +469,114 @@ def prepare_test_env(self):
+                    operations_classes = [op_name for op_name in dir(module_to_generate.operations)
+                                          if op_name[0].isupper()]
+                    break
+
+        for op_name in operations_classes:
+            op_cls = getattr(module_to_generate.operations, op_name)
+            for op_attr_name in dir(op_cls):
+                op_attr = getattr(op_cls, op_attr_name)
+                if isinstance(op_attr, types.FunctionType) and not op_attr_name.startswith("_"):
+                    func_content = self.get_function(op_attr)
+                    if not func_content:
+                        continue
+                    if operations_info.get(op_name):
+                        operations_info[op_name].append(func_content)
+                    else:
+                        operations_info[op_name] = [func_content]
+        tests_info = {'client': client_name, 'operations': operations_info}
+        return tests_info
+
+    def config_test(self, test_path, template_path):
+        if not Path(f'{test_path}/conftest.py').exists():
+            with open(template_path/'conftest.py', 'r') as fr:
+                content = fr.read()
+            with open(f'{test_path}/conftest.py', 'w') as fw:
+                fw.write(content)
+
+        # config ci.yml
+        ci_path = f'{Path(self.sdk_code_path()).parent}/ci.yml'
+        new_lines = ''
+        with open(ci_path, 'r+') as f:
+            if 'TestProxy' not in f.read():
+                f.seek(0, 0)
+                for line in f:
+                    if 'ServiceDirectory' in line:
+                        line += '    TestProxy: true\n'
+                    new_lines += line
+
+                with open(ci_path, 'w+') as fw:
+                    fw.write(new_lines)
+
+    @return_origin_path
+    def prepare_tests(self):
+        os.chdir('../../../')
+        test_path = self.sdk_code_path() + '/tests'
+        if not Path(test_path).exists():
+            os.mkdir(test_path)
+        module_name = f'azure.mgmt.{self.package_name}'
+        tests_info = self.get_tests_info(module_name)
+        client_name = tests_info['client']
+        operations = tests_info['operations']
+
+        template_path = Path('scripts/auto_release/templates')
+        env = jinja2.Environment(loader=jinja2.FileSystemLoader(template_path))
+        temp = env.get_template('testcase.py')
+        for operation_name, functions in operations.items():
+            if operation_name != 'Operations':
+                operation_name = re.sub(r'([a-z])([A-Z])', r'\1_\2', operation_name)[:-11]
+            operation_name = operation_name.lower()
+            temp_out = temp.render(module=module_name,
+                                   package=self.package_name,
+                                   client=client_name,
+                                   operation=operation_name,
+                                   functions=functions
+                                   )
```

Comment:
Currently, `operation_name` equals `operation_group` name, functions equals operation which is misleading. Please use operation and operation_group as variable name